src/feature_types/feature.js

```diff
@@ -39,15 +43,19 @@ Feature.prototype.toGeoJSON = function() {
 };
 
 Feature.prototype.internal = function(mode) {
+  var properties = {
+    id: this.id,
+    meta: Constants.meta.FEATURE,
+    'meta:type': this.type,
+    active: Constants.activeStates.INACTIVE,
+    mode: mode
+  };
+  for (var name in this.properties) {
+    properties[name] = this.properties[name];
```

Comment:
@ChrisBellew I want to stay away from possible clashes for properties. What do you think about having all user properties prefixed with `user_`. My concern with the prefix option is that it means the user provided properties will appear to change their name when exported from Draw.